### PR TITLE
Introduce new main level quick links

### PR DESCRIPTION
**Pull request by rainemak**
This change introduces "Supported devices", "Releases", "User
experience", and "Forum" to the main page of the docs.

Signed-off-by: Raine Makelainen <raine.makelainen@jolla.com>

### CODE REVIEW

**Review by llewelld — APPROVED** [review]
This looks great. It might be worth adding in to the commit message about the fact you've also included an up-to-date list of all the releases, but all of my comments are super-minor, so I'm approving already.

File: index.md [hunk 1]
```diff
@@ -48,6 +48,46 @@ Sailfish OS is developed by <a href="https://jolla.com">Jolla</a> and powered by
...
+      <br/><br/>
+      <span class="landing-link">Learn more ›</span>
+    </div>
+  </div></a>
+  <a href="/Releases">
+  <div class="landing-blank">
+    <span class="landing-head">Releases</span>
+    <div  class="landing-text landing-box-text">
+      Public Sailfish OS releases.
+      <br/><br/>
```
Comment (sledges) [inline index.md:66]: To align with all other paragraphs on the same line.
```suggestion
      <br/><br/><br/>
```

**Review by sledges — CHANGES_REQUESTED** [review]
Mobile view (Sailfish OS Xperia 10 III) in landscape results in 3x2 grid followed by 1x2 (also the order is not what you expect).

Such breakage after adding more squares has always been anticipated, given how much cross-browser/mobile tailoring has been poured to this front page... Improvements welcome when time permits:)

Couple of minor suggestions below.

File: index.md (hunk 1 above)
Comment (rainemak) [reply index.md:66]: Will break also User Experience into two lines and align landing-box-texts accordingly.

**Comment by rainemak** [comment]
Will squash review changes after this iteration is reviewed

**Review by sledges — APPROVED** [review]
Layout grid now looks amazing!

However now there are paragraph spacing issues on mobile browsers (Sailfish Browser and on an older Chrome/webkit) in "Supported Devices" and "User Experience" boxes.
But can be dealt in another PR perhaps?

**Comment by rainemak** [comment]
> Layout grid now looks amazing!
> 
> However now there are paragraph spacing issues on mobile browsers (Sailfish Browser and on an older Chrome/webkit) in "Supported Devices" and "User Experience" boxes. But can be dealt in another PR perhaps?

Yes, let's deal in another PR.

**Comment by rainemak** [comment]
Squashed review findings into the first commit.

**Pull request merged**